spec/features/visitor/visitor_can_create_an_account_spec.rb

```diff
@@ -0,0 +1,35 @@
+require 'rails_helper'
+
+
+RSpec.feature "Visitor can create an account" do
+  context "As a visitor not in database" do
+
+    it "visitor can create an account" do
+      Capybara.app = GrabBag::Application
+      stub_oauth
+
+      visit '/'
+
+      click_on "Login with Facebook"
+
+      expect(current_path).to eq('/sign_up')
+
```

Comment:
We will have to have the user pick a username as well (since username is in our routes)